src/sdk/model/o365/__json__/o365-job-json.ts

```diff
@@ -0,0 +1,83 @@
+import { EntityJson } from '../../common/__json__/entity-json';
+
+/**
+ * Interface for O365 Job JSON properties
+ */
+export interface O365JobJson extends EntityJson {
+  location_id: string;
+  o365_organization_uuid: string;
+  description: string;
+  last_run: number;
+  next_run: number;
+  enabled: boolean;
+  backup_type: O365JobBackupType;
+  last_status: O365JobStatus;
+  schedule_policy: O365JobSchedulePolicy;
+  type_vex: boolean;
+  type_vesp: boolean;
+  type_veod: boolean;
+}
+
+/**
+ * O365 Job Backup Type JSON enum
+ */
+export enum O365JobBackupType {
+  SELECTED_ITEMS = 'SelectedItems',
+  ENTIRE_ORGANIZATION = 'EntireOrganization'
+}
+
+/**
+ * O365 Job status JSON type
+ */
+export type O365JobStatus = 'Stopped' | 'Running' | 'Success' | 'Warning' | 'Failed';
```

Comment:
Just wanted to make sure. This and enum O365JobSessionStatus are two different things?

Reply:
They are. They're separated on the API so we separate them here too